document/fr.opensagres.xdocreport.document/src/main/java/fr/opensagres/xdocreport/document/preprocessor/sax/TransformedBufferedDocumentContentHandler.java

```diff
@@ -487,7 +482,7 @@ public void doEndElement( String uri, String localName, String name )
 
     public FieldMetadata getFieldAsTextStyling( String content )
     {
-        if ( formatter != null && fieldsMetadata != null )
+        if ( formatter != null && fieldsMetadata != null && !SaxUtils.isInstruction(content))

```

Comment:
@kevinsdooapp could you move SaxUtils.isInstruction(content) into IDocumentFormatter#isInstruction please.